### PR TITLE
MEN-1805: Setting wifi ssid and password in an image

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: Title

Signed-off-by: Ole Petter <ole.orhagen@cfengine.com>

### CODE REVIEW

File: 01.Getting-started/04.Deploy-to-physical-devices/docs.md [hunk 1]
```diff
@@ -126,6 +126,14 @@ Gateway=$IP_OF_MENDER_SERVER_FROM_DEVICE
 
 ! If you have a static IP address setup for several devices, you need several disk images so each get different IP addresses.
 
+## Wifi connectivity
+
+The raspberrypi demo image comes with Wifi connectivity enabled by default, thus the only thing needed in order for your device to connect to your network is setting the correct `<ssid>` and `<password>` in the `wpa_supplicant-nl80211@wlan0.conf` file on your device. Therefore running:
+```bash
+mender-artifact cat <sdimg>:/etc/wpa_supplicant/wpa_supplicant-nl80211-wlan0.conf | sed 's#[@]MENDER_DEMO_WIFI_PASSKEY[@]#<password>#' | sed 's#[@]MENDER_DEMO_WIFI_PASSKEY[@]#<ssid>#' | mender-artifact cp <sdimg>:/etc/wpa_supplicant/wpa_supplicant-nl80211-wlan0.conf"
```
Comment (estenberg) [inline 01.Getting-started/04.Deploy-to-physical-devices/docs.md]: Would set these as shell variables first, so the user does not need to edit this long command.

E.g.

Set your ssid and password...
SSID="example-ssid"
PASSWORD="abc"

Then run the following command to insert your wifi into the wpa-supplicant configuration...
mender-artifact cat...

**Comment by estenberg** [comment]
Does this work for Hosted Mender images as well? If so it would be really great if we could update https://hosted.mender.io/ui/#/help/connecting-devices/provision-a-demo/raspberry-pi-3 as well.

**Comment by oleorhagen** [comment]
I'm not sure what the hosted mender images look like @estenberg, but if they are in the format that is handled by the mender artifact tool, it should work. What do the mender images look like @maciejmrowiec ?

**Comment by estenberg** [comment]
good! @oleorhagen it is a separate task, but could you please update the Hosted Mender RPi3 help as well then? It is in the GUI repository.

**Comment by oleorhagen** [comment]
Sure: https://github.com/mendersoftware/gui/pull/363

**Comment by oleorhagen** [comment]
Btw, Is this ready to be merged?

**Pull request merged**